src/libaas/CMakeLists.txt

```diff
@@ -3,29 +3,24 @@
 ########################################
 
 set (BASYX_AAS_LIB_SHORTNAME "AAS")
-
 set (BASYX_AAS_LIBRARY_NAME "${PROJECT_SHORTNAME}${BASYX_AAS_LIB_SHORTNAME}")
-
 set (BASYX_AAS_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
 
-add_library(${BASYX_AAS_LIB_SHORTNAME})
+include(sources.cmake)
+
+add_library(${BASYX_AAS_LIB_SHORTNAME} ${SOURCE_FILES_BASYX} ${HEADER_FILES_BASYX})
```

Comment:
This does not satisfy the the install rountine, **target_sources()** has to stay imho.